data/autoyast_sle15/pc_tools.xml

```diff
@@ -116,8 +113,8 @@
                         sed -i 's/splash=silent\ quiet//' /etc/default/grub
                         sed -i '/GRUB_DISABLE_LINUX_UUID/s/^# //' /etc/default/grub
                         grub2-mkconfig -o /boot/grub2/grub.cfg
-                        cd /etc/alternatives/
-                        ln -fs /usr/bin/python3 python
+                        update-alternatives --install /usr/bin/python python /usr/bin/python2 1
+                        update-alternatives --install /usr/bin/python python /usr/bin/python3 2
```

Comment:
I guess, 1 refers to v2.7 and 2 to v3.5, right?

Reply:
1 and 2 are the priorities, 2 is more important so auto mode set default python version to 3.
I'm using the `/usr/bin/python2` and `/usr/bin/python3` links to avoid hardcoded the version, because it can change. In SLE15, `python2` is a link to `python2.7` and `python3` is a link to `python3.6`.